grunttasks/json2html.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
+
+// Four steps are performed to lint the files:
+//  1. Clean the directories
+//  2. Generate the JSON files from the PO files
+//  3. The JSON files are then written out to temporary HTML files
+//  4. The HTML files are linted
```

Comment:
Can the HTML linter just take the JSON strings instead of this intermediate representation of HTML files?